CMakeLists.txt

```diff
@@ -93,15 +93,20 @@ endif()
 include(QuickCppLibCacheLibrarySources)
 
 # Make available this library for install and export
+include(GNUInstallDirs)
+set(PROJECT_CONFIG_TEMPLATE_PATH "${CMAKE_CURRENT_LIST_DIR}/cmake/quickcpplibConfig.cmake.in")
+set(quickcpplib_CMAKELIB_DIR "${CMAKE_INSTALL_DATADIR}/cmakelib")
+set(quickcpplib_SCRIPTS_DIR "${CMAKE_INSTALL_DATADIR}/scripts")
+set(PROJECT_PACKAGE_PATH_VARS quickcpplib_CMAKELIB_DIR quickcpplib_SCRIPTS_DIR)
 include(QuickCppLibMakeInstall)
 install(DIRECTORY
-  "${CMAKE_CURRENT_SOURCE_DIR}/cmakelib"
+  "${CMAKE_CURRENT_SOURCE_DIR}/cmakelib/"
   COMPONENT headers
-  DESTINATION "${CMAKE_INSTALL_DATADIR}"
+  DESTINATION "${quickcpplib_CMAKELIB_DIR}"
```

Comment:
You sure this doesn't install into "${CMAKE_INSTALL_DATADIR}/cmakelib/cmakelib"?

Reply:
> The last component of each directory name is appended to the destination directory but a trailing slash may be used to avoid this because it leaves the last component empty.

[cmake docs#`install(DIRECTORY)`](https://cmake.org/cmake/help/latest/command/install.html#directory)

I added said trailing slash (see above). I tested these changes by locally updating the vcpkg ports. I'll open a vcpkg PR with those updates after this PR and the llfio PR get merged into master.